server/tweet/index.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const serie = require('fastseries')()
+
+const {
+  tweet: tweetSchema
+} = require('./schemas')
+const TweetService = require('./TweetService')
+
+module.exports = function (fastify, opts, next) {
+  // See user/index.js for some little explainations
+  serie(
+    fastify,
+    [
+      registerEnv,
+      registerMongo,
+      decorateWithTweetCollection,
+      registerMongoSetup,
+      decorateWithTweetService,
+      decorateWithUserClient,
+      registerRoutes
+    ],
+    opts,
+    next
+  )
```

Comment:
there is no need to use series here, you can just register them synchronously, it should all work and be taken care of.

Reply:
I cannot find any documentation on this. But ok